### PR TITLE
Fixes #397

### LINKED ISSUE #397: Link to *.md files in the docs are broken
Pre-Report Checklist
--------------------

- [x] I am running the latest version of mitiq
- [x] I checked to make sure that this bug has not already been reported

Issue Description
-----------------
On both of latest and stable releases of the docs page, the links to the [code of conduct](https://github.com/unitaryfund/mitiq/blob/master/CODE_OF_CONDUCT.md) and the links in contributing to docs section here are broken: https://mitiq.readthedocs.io/en/stable/contributing.html#updating-the-documentation.
Likely a regression of https://github.com/unitaryfund/mitiq/issues/211 

How to Reproduce
----------------

Visit https://mitiq.readthedocs.io/en/stable/contributing.html and try clicking on the code of conduct link at the bottom of the page.

### PR DESCRIPTION
**Pull request by erkska**
Fixes #397

Description
-----------
Fixed the bug where links didn't work.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ development of the library, from your local clone of the fork, run
 If you add new features to a function or class, it is required to add tests for such object. Mitiq uses a nested structure for packaging tests in directories named `tests` at the same level of each module.
 
 ### Updating the documentation
-Follow the guidelines in the Contributing to docs [instructions](https://mitiq.readthedocs.io/en/latest/read_README-docs.html) (look here on [GitHub](https://github.com/unitaryfund/mitiq/blob/master/docs/README-docs.md)), which include guidelines about updating the API-doc list of modules and writing examples in the users guide.
+Follow the guidelines in the Contributing to docs [instructions](https://mitiq.readthedocs.io/en/latest/README.html) (look here on [GitHub](https://github.com/unitaryfund/mitiq/blob/master/README.md)), which include guidelines about updating the API-doc list of modules and writing examples in the users guide.
```
Comment (rmlarose) [inline CONTRIBUTING.md:53]:
```suggestion
Follow these [instructions for contributing to the documentation](https://mitiq.readthedocs.io/en/latest/contributing_docs.html) which include guidelines about updating the API-doc list of modules and writing examples in the users guide.
```

**Review by rmlarose — CHANGES_REQUESTED** [review]
Thanks @erkska. One wrong link and I suggested a change to our previous wording.

**Comment by erkska** [comment]
Commited that change, whats the deal with wrong link?

**Review by rmlarose — APPROVED** [review]
Thanks! The link went to the readme when it should have gone to the contribution guidelines.

**Comment by crazy4pi314** [comment]
@all-contributors please add @LogMoss for doc, bug

**Pull request merged**